design-patterns/gc-intro/index.json-ld

```diff
@@ -94,38 +93,66 @@
 		},
 		{
 			"name": {
-				"en": "Introduction block with image",
-				"fr": "Bloc d'introduction avec image"
+				"en": "Introduction block with full-width image",
+				"fr": "Bloc d'introduction avec image pleine largeur"
 			},
 			"description": {
-				"en": "The introduction block pattern introduces the content of a landing page. This variation can contain an image and a super task button.",
-				"fr": "La configuration de conception du bloc d’introduction introduit le contenu d’une page d’accueil. Cette variation peur contenir une image."
+				"en": "The introduction block pattern introduces the content of a landing page. This variation contains an image.",
+				"fr": "La configuration de conception du bloc d’introduction introduit le contenu d’une page d’accueil. Cette variation contient une image."
 			},
 			"guidance": {
 				"en": "https://design.canada.ca/common-design-patterns/intro-block.html",
 				"fr": "https://conception.canada.ca/configurations-conception-communes/block-introduction.html"
 			},
 			"example": [
 				{
-					"en": { "href": "gc-intro-image-en.html", "text": "Introduction block with image" },
-					"fr": { "href": "gc-intro-image-fr.html", "text": "Bloc d'introduction avec image" }
+					"en": { "href": "gc-intro-full-width-image-en.html", "text": "Introduction block with full-width image" },
+					"fr": { "href": "gc-intro-full-width-image-fr.html", "text": "Bloc d'introduction avec image pleine largeur" }
 				}
 			],
 			"implementation": [
-				"_:implement_intro_image"
+				"_:implement_intro_full_width_image"
 			],
 			"history": [
 				{
 					"en": "March 2024 - Initial implementation of the variation.",
 					"fr": "Mars 2024 - Implémentation initiale de la variante."
 				}
 			]
+		},
+		{
+			"name": {
+				"en": "Introduction block with half-width image",
+				"fr": "Bloc d'introduction avec image demi-largeur"
+			},
+			"description": {
+				"en": "The introduction block pattern introduces the content of a landing page. This variation contains an image.",
+				"fr": "La configuration de conception du bloc d’introduction introduit le contenu d’une page d’accueil. Cette variation contient une image."
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/common-design-patterns/intro-block.html",
```

Comment:
Question for DTO: Do they want us to apply the fading out middle transition by using CSS or should we let the author to do that by using image editing tool?

Reply:
Not relevant anymore as DTO decided to use the current intro block pattern for the Topic template.